### ISSUE TITLE
Uncaught Error: ENOENT, open 'c:\wamp\www\themonastery.org\tests\NavigationLinksTest.php'

### ISSUE THREAD

**Issue by michaeljcalkins** [atom/atom#5065]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.166.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: ENOENT, open 'c:\wamp\www\themonastery.org\tests\NavigationLinksTest.php'

```
At c:\Users\Michael J. Calkins\AppData\Local\atom\app-0.166.0\resources\app\node_modules\q\q.js:126

Error: ENOENT, open 'c:\wamp\www\themonastery.org\tests\NavigationLinksTest.php'
  at Error (native)

```

### Commands

```
     -2:03.3 merge-conflicts:detect (atom-text-editor.editor.conflicted)
  3x -0:50.4 core:cut (atom-text-editor.editor)
     -0:48.6 core:save (atom-text-editor.editor)
     -0:26.4 core:cut (atom-text-editor.editor.conflicted.is-focused)
     -0:22.1 core:undo (atom-text-editor.editor.conflicted.is-focused)
  3x -0:17.0 bracket-matcher:go-to-matching-bracket (atom-text-editor.editor.conflicted.is-focused)
  2x -0:07.7 core:cut (atom-text-editor.editor.conflicted.is-focused)
  2x -0:06.4 core:undo (atom-text-editor.editor.conflicted.is-focused)
     -0:00.0 core:redo (atom-text-editor.editor.conflicted.is-focused)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      ".git",
      ".idea"
    ],
    "disabledPackages": [
      "autocomplete",
      "language-gfm",
      "language-coffee-script",
      "language-clojure",
      "language-c",
      "language-go",
      "language-java",
      "language-make",
      "language-mustache",
      "language-objective-c",
      "language-python",
      "language-ruby",
      "language-ruby-on-rails"
    ],
    "themes": [
      "atom-dark-ui",
      "monokai"
    ],
    "projectHome": "c:\\wamp\\www\\themonastery.org",
    "audioBeep": false,
    "autoHideMenuBar": true
  },
  "editor": {
    "fontFamily": "Consolas",
    "fontSize": 17,
    "lineHeight": 1.6,
    "autoIndentOnPaste": false,
    "preferredLineLength": 120,
    "tabLength": 4,
    "softWrap": true,
    "softTabs": false,
    "scrollPastEnd": true,
    "invisibles": {},
    "zoomFontWhenCtrlScrolling": false
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.4
autocomplete-plus, v1.1.0
color-picker, v1.2.6
editor-stats, v0.16.0
editorconfig, v0.3.0
highlight-line, v0.9.3
html-helper, v0.2.3
less-than-slash, v0.2.0
linter, v0.10.0
linter-csslint, v0.0.11
linter-jshint, v0.1.0
linter-php, v0.0.11
local-history, v2.0.0
merge-conflicts, v1.2.8
monokai, v0.9.0
open-last-project, v0.1.1
project-manager, v1.14.1

# Dev
No dev packages
```

**Comment by kevinsawicki** [comment]
Do you remember what you were doing when this happened?

**Comment by michaeljcalkins** [comment]
I was using the merge conflict package to resolve this:
(after I had removed that package)
![image](https://cloud.githubusercontent.com/assets/8272012/5748226/2b7fbb4c-9bf3-11e4-933a-49b0c6d91db2.png)

**Comment by kevinsawicki** [comment]
So it popped up when you saved the file?

**Comment by michaeljcalkins** [comment]
Sorry no it happened when I was scrolling down.  Atom froze and I got the error.  I hadn't clicked anything yet.
Could be related to: https://github.com/abe33/atom-color-highlight/issues/61
because I got that one too soon after.

**Comment by kevinsawicki** [comment]
This should be fixed by #4906 since it catches this error and shows a better message when it occurs in files opened in Atom.

Currently it is not possible to tell if it is coming from core or from a package given the stack trace it generates.

So I'm going to close this out and once #4906 is out it should provide better details.

**Comment by michaeljcalkins** [comment]
I'll report more if I can get it too happen again.  For now I removed the package I think was causing this too happen.
